### PR TITLE
Feature/load all

### PR DESCRIPTION
**Pull request by clumsyjedi**

### CODE REVIEW

File: src/clojure/clj_yaml/core.clj [hunk 1]
```diff
@@ -149,22 +149,34 @@
...
+            string
+            load-all? keywords))
 
 ;; From https://github.com/metosin/muuntaja/pull/94/files
 (defn generate-stream
   "Dump the content of data as yaml into writer."
   [writer data & opts]
   (.dump ^Yaml (apply make-yaml opts) (encode data) writer))
 
+(defn- load-stream [^Yaml yaml ^java.io.Reader input load-all? keywords]
```
Comment (slipset) [inline src/clojure/clj_yaml/core.clj:173]: This looks a lot like `load-str`, maybe they can be implemented in terms of each other?
By creating a `Reader` from a `String` via the `StringReader` you might just be able to eliminate `load-str` ?

Comment (clumsyjedi) [reply src/clojure/clj_yaml/core.clj]: Fair play, patch for this in f2d68bb

**Comment by borkdude** [comment]
@clumsyjedi Sorry we didn't get back to you. I've noticed this PR just now. Would you mind cleaning up the changes to `project.clj` and the `.lein_repl_history` file?/

/cc @lread

**Comment by lread** [comment]
I will carry out recommended changes.

**Pull request merged**